Merge main-v0.13.1 into main

barak-b-starkware opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/starkware-libs/blockifier/1521)
<!-- Reviewable:end -->

barak-b-starkware reviewed (commented):
> Reviewed 38 of 38 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/blockifier/1521)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @barak-b-starkware)



<!-- Sent from Reviewable.io -->

elintul reviewed (approved):
> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/blockifier/1521)* status: all files reviewed, 1 unresolved discussion (waiting on @barak-b-starkware and @Yoni-Starkware)

___
*[`crates/blockifier/src/transaction/transactions_test.rs` line 368 at r1](https://reviewable.io/reviews/starkware-libs/blockifier/1521#-NqhMXrj-ZTDm19MWDgF:-NqhMXrk0siQALN-3HXV:b-n6yzzq) ([raw file](https://github.com/starkware-libs/blockifier/blob/82eecedfdade8cdfde145996b2158dafcd542f36/crates/blockifier/src/transaction/transactions_test.rs#L368)):*
> ```XML
>         signature: TransactionSignature::default(),
>         nonce: Nonce::default(),
>         sender_address: account_contract_address,
> ```

Where was this moved to?


<!-- Sent from Reviewable.io -->

barak-b-starkware commented:
*[`crates/blockifier/src/transaction/transactions_test.rs` line 368 at r1](https://reviewable.io/reviews/starkware-libs/blockifier/1521#-NqhMXrj-ZTDm19MWDgF:-NqhNi7kCijR0TLfnIAz:b-qwf13x) ([raw file](https://github.com/starkware-libs/blockifier/blob/82eecedfdade8cdfde145996b2158dafcd542f36/crates/blockifier/src/transaction/transactions_test.rs#L368)):*
<details><summary><i>Previously, elintul (Elin) wrote…</i></summary><blockquote>

Where was this moved to?
</blockquote></details>

Removed and replaced with `create_trivial_calldata` in `test_utils.py`.


<!-- Sent from Reviewable.io -->
